src/agent/config.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * @typedef {object} DebugAgentConfig
+ */
+
+module.exports = {
+  /**
+   * @property {boolean=} Whether the debug agent should be started.
```

Comment:
I always understood the `=` syntax to indicate an optional parameter to a function. I don't think it makes sense for properties to be "optional". Maybe nullable `?` is better though I'm not a fan of a nullable boolean?